package.json

```diff
@@ -108,9 +109,10 @@
     "react": "16.8.3",
     "react-docgen-typescript": "1.12.3",
     "react-dom": "16.8.3",
-    "react-styleguidist": "8.0.6",
+    "react-styleguidist": "9.0.1",
     "rimraf": "2.6.3",
     "source-map-loader": "0.2.4",
+    "style-loader": "0.23.1",
```

Comment:
See comment above.

Reply:
Yes. 'module not found' errors where thrown before.